src/In.ProjectEKA.HipService/appsettings.Development.json

```diff
@@ -19,7 +19,14 @@
     "BaseUrl":"http://otp"
   },
   "rabbit": {
-    "HostName": "rabbitmq",
+    "HostName": "localhost",
     "Port": 5672
+  },
+  "dataFlow": {
+    "dataSizeLimitInMbs": 5,
+    "dataLinkTTLInMinutes": 10
+  },
+  "hip": {
+    "url": "http://localhost:8000"
   }
```

Comment:
HIU will use this URL right to pull? then it needs to be a public URL, have to think about it.